pusher/pusher.xcodeproj/project.pbxproj

```diff
@@ -11,10 +11,12 @@
 		3A590CE02325E42E005AA8BA /* Application.xib in Resources */ = {isa = PBXBuildFile; fileRef = 3A590CDF2325E42E005AA8BA /* Application.xib */; };
 		3ACF98FD2320E1BC00C40BB4 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3ACF98FC2320E1BC00C40BB4 /* AppDelegate.swift */; };
 		3ACF99012320E1BD00C40BB4 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 3ACF99002320E1BD00C40BB4 /* Assets.xcassets */; };
+		C33DC8BCEF7ABD522D62DFB4 /* Pods_pusher.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 27E8F9EE3A5F9B9E5E773FB1 /* Pods_pusher.framework */; };
```

Comment:
@KittenYang Please remove all the modifications made by `pod install`.

Reply:
@malayli  All Done!